examples/GCL/augmentors/augmentor.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import List, NamedTuple, Optional, Tuple
+
+import torch
+from torch_sparse import SparseTensor
+
+
+class Graph(NamedTuple):
```

Comment:
Lets not create a new data object. Reuse `torch_geometric.data.Data`.

Reply:
Thank you for advices. I will check the code!